### ISSUE TITLE
Generic shop locations

### ISSUE THREAD

**Issue by HellMapGoesCoding** [OSMLatvija/Osmalyzer#4]
Non-grocery shops. Currently, not parsing any yet. Logic would be pretty much identical to #3. Need to compile a list of potentials.

**Comment by markalex2209** [comment]
What about Fixprice https://lv.fix-price.com/stores?

**Comment by markalex2209** [comment]
Also, shops that are not groceries, and seemingly have extensive networks: alcohol shops like rigas balzams, spirits and wine, alkoutlet. And sweetshops: laima, skriveru. 
Not sure how many of them are, or if the have maps.

**Comment by HellMapGoesCoding** [comment]
> Not sure how many of them are, or if the have maps.

Yeah, there are a ton of different shops. I haven't added any to the list, because I was looking at their maps and whether I can parse it first before adding it to the list as "todo" or "can't do". This is why I haven't added all these.

**Comment by markalex2209** [comment]
Consider creating a discussion, we're one could put a brand for future. That way it'll be possible to later see what was already considered and if rejected - why.

**Comment by HellMapGoesCoding** [comment]
> Consider creating a discussion, we're one could put a brand for future. That way it'll be possible to later see what was already considered and if rejected - why.

Oh, nothing has been rejected, so to speak. I have listed the ones I failed to parse or get a map for, which is very few because I hadn't looked through most of the shops, just the big/popular ones. And, so far, almost all have had a map and only a few (like Lidl) had a directly unparsable map (or at least I couldn't be bothered to spend so long on doing it).

So I guess anyone can just mention brands in the issue(s) and I can add it to the list. I personally didn't want to spend time adding entries just by name without linking to at least their map or stating that I couldn't find one. But I guess nothing is stopping us from making a "raw" list.

**Comment by markalex2209** [comment]
A bunch more candidates:
- Narvessen: https://narvesen.lv/kontakti/
- Banknote: https://veikals.banknote.lv/filiales
- MyFitness: https://www.myfitness.lv/ru/clubs/kluby/. Only has 13 location, not sure if worth it. And also obviously not a shop)

**Comment by markalex2209** [comment]
And more:
- casino [Olympic](https://olympic-casino.lv/ru/kontakty)
- casino [Fenikss](https://www.feniksscasino.lv/lv/venues)
- [Vita Lombards](http://vitalombards.lv/ru/kontakti/)

**Comment by markalex2209** [comment]
There is a list and map of farm market places here: https://gadatirgi.lv/bralutirgus/
Don't know how useful this is, since it seems like there are some one-time events shown too. 
Also, no idea on how authoritative this list is.
